CardanoSharp.Koios.Client/CardanoSharp.Koios.Client.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.1</TargetFramework>
-        <Nullable>enable</Nullable>
-        <Version>1.0.9</Version>
+        <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
@nothingalike This should open up the package to more .NET frameworks

Reply:
yeah my currently opened PR is just adding netstandard2.0 along side 2.1.